docs/customization/menu.rst

```diff
@@ -3,34 +3,33 @@ Customizing Menus
 
 Adding new positions in your menu is done **via events**.
 
-You have got the ``Sylius\Bundle\WebBundle\Event\MenuBuilderEvent`` with ``FactoryInterface`` and ``ItemInterface`` of `KnpMenu`_. So you can manipulate the whole menu.
+You have got the ``Sylius\Bundle\UiBundle\Menu\Event\MenuBuilderEvent`` with ``FactoryInterface`` and ``ItemInterface`` of `KnpMenu`_, this lets yoU manipulate the whole menu.
 
-You've got such events that you should be subscribing to:
+You've got two events that you should be subscribing to:
 
 .. code-block:: php
 
-    sylius.menu.admin.main                  # Admin Panel menu
-    sylius.menu_builder.frontend.main       # Main Shop menu (top bar)
-    sylius.menu_builder.frontend.currency   #
-    sylius.menu_builder.frontend.taxons     #
-    sylius.menu_builder.frontend.social     #
-    sylius.menu_builder.frontend.account    #
+    sylius.menu.shop.account # For the menu of the MyAccount section in shop
+    sylius.menu.admin.main # For the Admin Panel menu
 
-How to customize a Menu?
-------------------------
+How to customize Admin Menu?
+----------------------------
 
-1. In order to add items to any of the Menus in **Sylius** you have to create a ``AppBundle\EventListener\MenuBuilderListener`` class.
+.. tip::
 
-In the example below we are adding a one new item to the Admin panel menu and a one new item to main menu of the shop.
+    Admin Panel menu is the one in the left expandable sidebar on the ``/admin/`` url.
+
+**1.** In order to add items to the Admin menu in **Sylius** you have to create a ``AppBundle\EventListener\Menu\Admin\MenuBuilderListener`` class.
```

Comment:
Looks good to me, but previously we used to have listeners. Therefore if somebody is already customizing menus, they probably have Listeners. Then they'll just change the implementation not the whole class.